app/infrastructure/mailing_service/models/base_models.py

```
@@ -0,0 +1,81 @@
+	seen: set[str] = set()
+	out: list[EmailStr] = []
+	for e in seq or []:
+		k = normalize_email(e)
+		if k in seen:
+			continue
+		seen.add(k)
+		out.append(e)
+	return out
+
+class EmailEnvelope(BaseModel):
+	subject: NonBlankStr = Field(default=..., description="The email’s title (what shows in the inbox). Keep it short and human (avoid ALL CAPS/spammy words).")
```

Ruff (0.13.1)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)